### ISSUE TITLE
Distinct responses for the same user when requesting los via current projection

### ISSUE THREAD

**Issue by bastianjoel** [OpenSlides/openslides-autoupdate-service#666]
Although the title may sound quite specific I am not entirely sure if this only happens for the described setup.

**Setup**: Create assignment, project it to the reference projector and add speakers to its list of speakers. 

With the following request data is missing for list of speakers with about every second request
[request.txt](https://github.com/OpenSlides/openslides-autoupdate-service/files/11259972/request.txt)

To send this request I used the performance tool with a delegate.
```
openslides-performance request -d <url> -u m1dummy1 -p pass /system/autoupdate?single=1 -b <REQUEST> | grep -q 'list_of_speakers/<los id>/meeting_id' && echo 'contains los'
```

Example response with los missing:
[not_contains_los.txt](https://github.com/OpenSlides/openslides-autoupdate-service/files/11259997/not_contains_los.txt)

Example response containing los:
[contains_los.txt](https://github.com/OpenSlides/openslides-autoupdate-service/files/11260000/contains_los.txt)

**Comment by ostcar** [comment]
I have to test this.

Did you use the current main of the autoupdate-service or a version before https://github.com/OpenSlides/openslides-autoupdate-service/pull/662

Because this issue could be fixed with this PR (hard to tell, since I am not sure why this happens).

**Comment by ostcar** [comment]
I can reproduce this with a commit before #662 but not with the current main. I think this is fixed. If it happens again, then please reopen this issue.

The reason for this issue is, that it requests the following fields:

```
meeting/1/reference_projector_id/current_projection_ids/content_object_id: {...}
meeting/1/projector_ids/current_projection_ids/content_object_id: null
```

Since the reference_projector is also in `projector_ids`, its content_object_id is requested twice. One time with `null` and one time with `{"type": "generic-relation"}`. Both of this fields where saved in a hashmap with the same key. So the second write replace the first entry. This was fixed with #662

**Comment by bastianjoel** [comment]
Works for me with current main too!
